.github/workflows/test.yml

```diff
@@ -85,11 +85,31 @@ jobs:
 
       - name: Upload Report PDFs
         if: always()
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         with:
           name: example reports
           path: test-report-*.pdf
 
+  test-library-import-bare:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.12'
+      - name: Install dependencies in a venv
+        run: |
+          python3 -m venv venv
+          . venv/bin/activate
+          pip install -q -U pip
+          pip install --progress-bar=off .
+
+      - name: Test import speeds
```

Comment:
The import speeds aren't being tested, maybe a copy/paste error?

Reply:
Yep. Thanks